qcfractal/storage_sockets/sqlalchemy_socket.py

```diff
@@ -1184,25 +1196,36 @@ def update_results(self, record_list: List[ResultRecord]):
         -------
             number of records updated
         """
+        query_ids = [res.id for res in record_list]
+        duplicates = len (query_ids) != len (set(query_ids))
```

Comment:
Please use the black formatter, `make format` will automatically apply this.

Reply:
Formatted the code.